numba_dpex/core/runtime/_dpexrt_python.c

```diff
@@ -1157,6 +1146,11 @@ static int DPEXRT_sycl_queue_from_python(PyObject *obj,
         goto error;
     }
 
+    DPEXRT_DEBUG(DPCTLSyclDeviceRef device_ref;
```

Comment:
Might be useful to convert into a helper function that may be used other places too.